Core/Core.cpp

```diff
@@ -150,7 +150,12 @@ static bool IsWindowSmall(int pixelWidth, int pixelHeight) {
 // TODO: Feels like this belongs elsewhere.
 bool UpdateScreenScale(int width, int height) {
 	bool smallWindow;
-#if PPSSPP_PLATFORM(WINDOWS) && !PPSSPP_PLATFORM(UWP)
+#if defined(USING_QT_UI)
+	g_dpi = (float)System_GetPropertyInt(SYSPROP_DISPLAY_DPI);
+	float g_logical_dpi = (float)System_GetPropertyInt(SYSPROP_DISPLAY_LOGICAL_DPI);
+	g_dpi_scale_x = g_logical_dpi / g_dpi;
+	g_dpi_scale_y = g_logical_dpi / g_dpi;
```

Comment:
I tries to get the native scale factor. This should work on Windows too ( untested )